### ISSUE TITLE
Values in AllModesEnumeration vs ContinuousModeEnumeration

### ISSUE THREAD

**Issue by skinkie** [NeTEx-CEN/NeTEx#45]
In ContiniousModes we are using walk and cycle. In AllModesEnumeration we are using foot and bicycle. This should be made more consistent.

**Comment by Aurige** [comment]
Yes, as explained in Brussels on Nov 12th, there will be very soon an extension for new modes where we should be able to manage this.

**Comment by ue71603** [comment]
Seems it was notsolved in new modes.  
@Aurige  How do we continue?

	<xsd:simpleType name="ContinuousModeEnumeration">
		<xsd:annotation>
			<xsd:documentation>Allowed values for continuous MODES of transport applicable to private and non-timetabled transport.</xsd:documentation>
		</xsd:annotation>
		<xsd:restriction base="xsd:NMTOKEN">
			<xsd:enumeration value="walk"/>
			<xsd:enumeration value="car"/>
			<xsd:enumeration value="taxi"/>
			<xsd:enumeration value="cycle"/>
			<xsd:enumeration value="drt"/>
			<xsd:enumeration value="movingWalkway"/>
			<xsd:enumeration value="through"/>
			<xsd:enumeration value="ski"/>
			<xsd:enumeration value="skate"/>
		</xsd:restriction>
	</xsd:simpleType>


	<xsd:simpleType name="AllModesEnumeration">
		<xsd:annotation>
			<xsd:documentation>Allowed values for r MODES: TPEG pti_table 01.</xsd:documentation>
		</xsd:annotation>
		<xsd:restriction base="xsd:NMTOKEN">
			<xsd:enumeration value="all"/>
			<xsd:enumeration value="unknown"/>
			<xsd:enumeration value="air"/>
			<xsd:enumeration value="bus"/>
			<xsd:enumeration value="trolleyBus"/>
			<xsd:enumeration value="tram"/>
			<xsd:enumeration value="coach"/>
			<xsd:enumeration value="rail"/>
			<xsd:enumeration value="intercityRail"/>
			<xsd:enumeration value="urbanRail"/>
			<xsd:enumeration value="metro"/>
			<xsd:enumeration value="water"/>
			<xsd:enumeration value="cableway"/>
			<xsd:enumeration value="funicular"/>
			<xsd:enumeration value="lift"/>
			<xsd:enumeration value="snowAndIce"/>
			<xsd:enumeration value="taxi"/>
			<xsd:enumeration value="selfDrive">
				<xsd:annotation>
					<xsd:documentation>See pti12_x.</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="foot"/>
			<xsd:enumeration value="bicycle"/>
			<xsd:enumeration value="motorcycle"/>
			<xsd:enumeration value="car"/>
			<xsd:enumeration value="shuttle"/>
			
			
		</xsd:restriction>
	</xsd:simpleType>

**Comment by Aurige** [comment]
do we have a ContinuousModeEnumeration in NeTEx ?

**Comment by ue71603** [comment]
It exists in the "next" branch, but it is absolutely nowhere used. What do we do? Delete it in next?

**Comment by skinkie** [comment]
Next may contain new stuff. but if we already know it should be removed, lets do so.

**Comment by trurlurl** [comment]
`ContinuousModeEnumeration` has been removed from the code in the meantime, some spooky day in november - see netex_mode_support.xsd:
```
<Date><Modified>27-11-26</Modified>New Modes;
  Drop ContinuousModes 
</Date>
```
So, this problem is solved. 

Also, enum value `walk` doesn't occur in the code any more, for `cycle` see below. Here remains a little problem to be discussed and at least `pedalCycle` is awaiting a description. What is the use of `cycle`, would it be used to indicate that both `pedalCycle` and `eCycle` can be transported? Is it really needed?

```
<xsd:simpleType name="PortableVehicleCategoryEnumeration">
		<xsd:annotation>
			<xsd:documentation>Allowed values for Portable VEHICLE types. +V2.0</xsd:documentation>
		</xsd:annotation>
		<xsd:restriction base="xsd:string">
			<xsd:enumeration value="scooter"/>
			<xsd:enumeration value="eScooter"/>
			<xsd:enumeration value="cycle"/>
			<xsd:enumeration value="pedalCycle">
				<xsd:annotation>
					<xsd:documentation/>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="eCycle"/>`
```

**Comment by Aurige** [comment]
My understanding is that pedalCycle means non-motorised cycle ... 
- Cycle : all bicyles
- eCycle : only electric cycles
- pedalCycle : only non electric cycles

but most eCycle had pedals, so that's true that it is confusing.
to be validated by the group

**Comment by Aurige** [comment]
> @Aurige This makes sense in my view, too. I'm going to add the descriptions.
> 
> But one thing remains to be clarified: can / should we rename `cycle` to `bicycle`? This is the only occurrence of "cycle", and I think that typical uses of "bicycle" don't mind whether it is a conventional or a motorised bicycle.

If we want to do this, we need to deprecate "cycle" since it is already used ... not sure that's worth it provided that we add the proper annotation
